hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -43,89 +18,92 @@ This section describes all the configuration items available in the bootloader.
 
-Default: 1 (0 in original version of bootloader 2019-05-10)  
+Default: `1` (`0` in original version of bootloader 2019-05-10)  
 Version: All  
 
 ### POWER_OFF_ON_HALT
 
 If 1 and WAKE_ON_GPIO=0 then switch off all PMIC outputs in halt. This is lowest possible power state for halt but may cause problems with some HATs because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.
 
-Default: 0  
+Default: `0`  
 Version: 2019-07-15  
 
 ### BOOT_ORDER
 The BOOT_ORDER setting allows flexible configuration for the priority of different bootmodes. It is represented as 32bit unsigned integer where each nibble represents a bootmode. The bootmodes are attempted in lowest significant nibble to highest significant nibble order.
 
-E.g. 0x21 means try SD first followed by network boot then stop. Whereas 0x2 would mean try network boot and then stop without trying to boot from the SD card.
+E.g. `0x21` means try SD first followed by network boot then stop. Whereas 0x2 would mean try network boot and then stop without trying to boot from the SD card.
 
 The retry counters are reset when switching to the next boot mode.
 
-BOOT_ORDER fields  
+`BOOT_ORDER` fields  
 The BOOT_ORDER property defines the sequence for the different boot modes. It is read right to left and up to 8 digits may be defined.
 
-* 0x0 - NONE (stop with error pattern)
-* 0x1 - SD CARD
-* 0x2 - NETWORK
-* 0x3 - USB device boot - Reserved - Compute Module only.
-* 0x4 - USB mass storage boot (since 2020-09-03)
-* 0xf - RESTART (loop) - start again with the first boot order field. (since 2020-09-03)
+| Value | Mode      | Description                                                              |
+|-------|-----------|--------------------------------------------------------------------------|
+|  0x0  |  NONE     |  Stop and display error pattern                                          |
+|  0x1  |  SD CARD  |  SD card (or EMMC on Compute Module 4)                                   |
+|  0x2  |  NETWORK  |  Network boot                                                            |
+|  0x3  |  USB DEV  |  USB device boot - See [usbboot](https://github.com/raspberrypi/usbboot) |
+|  0x4  |  USB MSD  |  USB mass storage boot (since 2020-09-03)                                |
+|  0xf  |  RESTART  |  Start again with the first boot order field. (since 2020-09-03)         |
 
-Default: 0xf41 (0x1 in versions prior to 2020-09-03)  
+Default: `0xf41` (`0x1` in versions prior to 2020-09-03)  
 Version: 2020-04-16  
 
 * Boot mode `0x0` will retry the SD boot if the SD card detect pin indicates that the card has been inserted or replaced.
 * The default boot order is `0xf41` which means continuously try SD then USB mass storage.
+* USB device boot is intended for use with Compute Module 4 to load a debug image e.g. a Linux RAM disk. This should be the last boot option because it does not currently support timeouts or retries.
```

Comment:
Is "debug image" the correct phrasing here? Isn't USB device boot also used to flash the EMMC? (which is something that most users wouldn't class as "debug")

Reply:
specified that this a custom image i.e. something other than the usbboot MSD device mode feature